cmd/jx/main.go

```diff
@@ -5,34 +5,42 @@ import (
 	"os"
 
 	"foxygo.at/jsonnext"
+	jxkong "foxygo.at/jsonnext/kong"
 	"github.com/alecthomas/kong"
 	"github.com/google/go-jsonnet"
 )
 
-var CLI struct {
-	ImportPath []string `name:"jpath" sep:":" short:"J"`
-	Filename   string   `arg:"" optional:"" help:"File to evaluate. stdin is used if omitted or \"-\""`
+type config struct {
+	jxkong.Config
+	Filename string `arg:"" optional:"" help:"File to evaluate. stdin is used if omitted or \"-\""`
 }
 
 func main() {
-	kong.Parse(&CLI)
+	cli := parseCLI()
```

Comment:
I'd consider calling it `cfg` instead of `cli`, because it makes it more obvious that it's of type config.
feel free to ignore.

Reply:
I tried it, but there are too many configs - `cfg.Config.Config`. I will keep it called `cli` since that is what kong calls it.